.github/workflows/buildAndTest.yml

```diff
@@ -168,6 +168,8 @@ jobs:
           cmake .. \
             -GNinja \
             -DCMAKE_BUILD_TYPE=Debug \
+            -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
+            -DCMAKE_EXE_LINKER_FLAGS_INIT="-fuse-ld=lld" -DCMAKE_MODULE_LINKER_FLAGS_INIT="-fuse-ld=lld" -DCMAKE_SHARED_LINKER_FLAGS_INIT="-fuse-ld=lld" \
```

Comment:
LLVM_USE_LINKER isn't sufficient?

Reply:
I've been ever able to get `LLVM_USE_LINKER` to behave consistently across platforms/environments/projects.

Reply:
Interesting... my experience is that it always just worked..  has it not been working in the CI?

Reply:
No clue but you can see here https://github.com/Xilinx/mlir-aie/actions/runs/6999303703/job/19038425447 where I tried both `LLVM_USE_LINKER` and `LLVM_CCACHE_BUILD` and both "didn't take".